### ISSUE TITLE
No consistency in generated saliencies

### ISSUE THREAD

**Issue by nicenoize** [copenlu/xai-benchmark#3]
Hello! I noticed, that the values of the generated saliencies are quite different after generating them multiple times. 

While trying to find the origin of this issue i noticed: 
- in line 111 of interpret_grads_occ.py: executing model(batch[0], attention_mask=batch[1], labels=batch[2].long()) returns a different tensor after each call. I was expecting to receive the same results
- line 157: ablator.attribute(input_embeddings, target=cls_, additional_forward_args=additional) also returns different tensors with each call

I hope you can help me clarify this
